packages/clerk-js/src/ui/styledSystem/common.ts

```diff
@@ -194,12 +194,13 @@ const unstyledScrollbar = (t: InternalTheme) => ({
   '::-webkit-scrollbar-track': {
     background: 'transparent',
   },
+  scrollbarGutter: 'stable both-edges',
```

Comment:
I believe this is not available for Safari, will that be a problem ?

Reply:
The alternative is `overflow: overlay` which is a legacy value.